cmd/server/assets/realmadmin/show.html

```diff
@@ -23,7 +23,16 @@ <h1>Realm stats</h1>
     </p>
 
     <div class="card mb-3 shadow-sm">
-      <div class="card-header">Statistics</div>
+      <div class="card-header">Statistics
```

Comment:
I'm wondering what this would look like more like this:
https://gist.github.com/whaught/7ad634843cc4de808796bdce5cdd8a33

just to get things out of the 'card-header' div and have an all-encompassing body div

Reply:
I don't like any of the styling. Here's what mine looks like:

![image](https://user-images.githubusercontent.com/3063869/97601630-c2699480-19e0-11eb-8124-bc2d381047e1.png)

And yours

![image](https://user-images.githubusercontent.com/3063869/97601767-e927cb00-19e0-11eb-91d3-97bd3521b692.png)


I don't like any of them. I am happy to iterate till we're blue in the face, but perhaps you're the better man for the job here.

Reply:
/lgtm

Let's go with this, we can tweak later